src/geckomat/get_enzyme_data/getECfromGEM.m

```diff
@@ -1,6 +1,9 @@
-function model = getECfromGEM(model, ecRxns)
+function [model, invalidEC, invalidECpos] = getECfromGEM(model, ecRxns)
 % getECfromGEM
-%   Use the model.eccodes to populates the model.ec.eccodes field.
+%   Use the model.eccodes to populates the model.ec.eccodes field. EC
+%   numbers that are not formatted as four numbers separated by periods,
+%   possibly with trailing wildcards. Examples: 1.2.3.4 or 1.2.3.- while
+%   invalid EC numbers are 1.2.3 or 1_2_3_4.
```

Comment:
And also give an example: 1.2.3.4;1.2.3.5 is valid while 1.2.3.4|1.2.3.5 is invalid.

Reply:
```suggestion
%   invalid EC numbers are 1.2.3 or 1_2_3_4. Multiple EC numbers are
%   separated by ; for instance 1.2.3.4;1.2.3.5 not 1.2.3.4|1.2.3.5.
```